test/NetCore.Test/ExceptionTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Threading.Tasks;
+using Orleans;
+using Orleans.Hosting;
+using Orleans.Runtime.Configuration;
+using Xunit;
+
+namespace NetCore.Test
+{
+    [Trait("Category", "BVT")]
+    public class ExceptionTests
+    {
+        private readonly ISiloHost silo;
+        private readonly IClusterClient client;
+
+        public ExceptionTests()
+        {
+            this.silo = SiloHostBuilder.CreateDefault().AddApplicationPartsFromAppDomain().ConfigureLocalHostPrimarySilo().Build();
+            this.silo.StartAsync().GetAwaiter().GetResult();
```

Comment:
This is leaking silos and clients. Be sure to make the class IDisposable and always stop them